src/views/BrokeringRuns.vue

```diff
@@ -7,7 +7,7 @@
         <ion-buttons slot="end" v-if="brokeringGroups.length">
           <ion-button color="primary" @click="addNewRun">
             {{ translate("New Run") }}
-            <ion-icon :icon="addOutline" />
+            <ion-icon :icon="arrowForwardOutline" />
```

Comment:
Hi @coolnj4 

You didn't need to change this icon, its already addOutline, you need to change the icon of `Create brokering run` from `arrowForwardOutline` to `addOutline`.

Please check the related issue attached in the comment and let me know if you need any furthur assistance.